test/pkg/gitea/crd.go

```diff
@@ -11,7 +11,10 @@ import (
 )
 
 func createToken(topts *TestOpts) (string, error) {
-	token, _, err := topts.GiteaCNX.Client.CreateAccessToken(gitea.CreateAccessTokenOption{Name: topts.TargetNS})
+	token, _, err := topts.GiteaCNX.Client.CreateAccessToken(gitea.CreateAccessTokenOption{
+		Name:   topts.TargetNS,
+		Scopes: []string{"repo", "admin:org", "admin:public_key", "admin:repo_hook", "admin:org_hook", "notification", "user", "delete_repo", "package", "admin:gpg_key", "admin:application", "sudo"},
```

Comment:
Ah This is the hack which you were mentioning in slack related to gitea right

Reply:
not exactly,  the Scopes field is in go-gitea API but not in the structs library, so i had to modify the structs manually in vendor i have checked and they haven't updated it yet, will look out for this until they do